frame/support/src/storage/storage_noop_guard.rs

```diff
@@ -0,0 +1,113 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![cfg(test)] // Testing only since it can panic.
```

Comment:
`test` features are only enabled crate locally. Meaning any other crate can not use this type.

Reply:
So what do? Just remove it or is there a special folder for testing stuff?  
I could add a `feature = std` so that it cannot be included in the runtime.

Reply:
We normally write:

```
	#[cfg(any(feature = "std", feature = "runtime-benchmarks", test))]
```